metricbeat/etc/fields.yml

```diff
@@ -8,7 +8,28 @@ defaults:
   ignore_above: 1024
 
 sections:
+  - ["general", "General event data"]
   - ["redis-info", "Redis Info Stats"]
+
+general:
+  type: group
+  description: >
+    General fields for each event
+  fields:
+    - name: response_time
```

Comment:
Ah, right. Since it's just a number, no problem with ES and Kibana's ms limitation. I'm ++ on a flag to enable ns. I think ms is a good default and I'm certain that some folks will need more precision.